schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -1380,8 +1380,6 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Compute.Extensions.json#/resourceDefinitions/virtualMachines_extensions" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Compute.Extensions.json#/resourceDefinitions/virtualMachineScaleSets_extensions" },
                   { "$ref": "https://schema.management.azure.com/schemas/2019-03-01/Microsoft.Compute.json#/resourceDefinitions/availabilitySets" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-03-01/Microsoft.Compute.json#/resourceDefinitions/galleries_applications" },
```

Comment:
why you remove this?

Reply:
Gallery schemas were moved to Microsoft.Compute.Galleries.json from Microsoft.Compute.json

Reply:
The --tag=2019-07 removed these references in compute.json